src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/UserAccessServiceTest.kt

```diff
@@ -1495,7 +1497,7 @@ class UserAccessServiceTest {
   }
 
   @Nested
-  inner class UserCanWithdrawApplication {
+  inner class UserCanWithdrawAdpplication {
```

Comment:
I think you've introduced a tiny typo here

Reply:
oops! good spot. Fixed.